src/common/Utils.ts

```diff
@@ -36,13 +36,14 @@ export function event2topic (contract: any, names: any): any {
     .map(abi.encodeEventSignature)
 }
 
+//caller provider's eth_signTypedData
```

Comment:
I don't know why, but your comments actually make me understand less. Could you please stop putting comments in code?